functions/merge_sorted_lists.py

```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
+    merged = l1 + l2
+
+    return sorted(merged)
```

Comment:
great job, we may be able to dry it up by putting everything in the return statement
''' python
return sorted(l1 + l2)
'''

Reply:
Thanks, Gloria! I'll do that.